ui/src/app/applications/components/application-operation-state/application-operation-state.tsx

```diff
@@ -185,7 +185,12 @@ export const ApplicationOperationState: React.StatelessComponent<Props> = ({appl
                                             {resource.hookType}
                                         </div>
                                         <div className='columns large-4 small-8' title={resource.message}>
-                                            <div className='application-operation-state__message'>{resource.message}</div>
+                                            <div className='application-operation-state__message'>
+                                                {resource.message.includes('metadata.annotations: Too long: must have at most 262144 bytes')
```

Comment:
In my opinion it would be better to handle this error message and wrap it on backend side, so it will be consumed also in CLI and probably in other places

Reply:
hmm I would work on it as soon as i get time.

Reply:
@pasha-codefresh there was already a function on the backend for augmenting the messages coming from API but switch structure was designed for exact string match and this would need a substring match so instead of changing the switch expression format I added it as a default case with substring contains method.  now this works for both CLI and UI 

a SS for cmd  output

<img width="868" alt="Screenshot 2024-03-22 120128" src="https://github.com/argoproj/argo-cd/assets/91555602/41d626af-f470-4a85-9cf5-3bd07903267e">